app/scripts/services/tile-proxy.js

```diff
@@ -158,11 +124,37 @@ export const setTileProxyAuthHeader = (newHeader) => {
 export const getTileProxyAuthHeader = () => authHeader;
 
 /**
- * @param {RequestContext} req
+ * @param {Array<WrappedTilesRequest>} requests
+ * @param {Array<WrappedTilesRequest>} requests
+ */
+const bundleRequests = (requests) => {
```

Comment:
Can requests be bundled by anything other than id? If not I'd consider calling this bundleRequestsById to make it clearer what is happening.

Reply:
I added a docstring and a unit test